shell/platform/linux/fl_key_event_plugin.cc

```diff
@@ -41,12 +50,17 @@ static void fl_key_event_plugin_class_init(FlKeyEventPluginClass* klass) {
   G_OBJECT_CLASS(klass)->dispose = fl_key_event_plugin_dispose;
 }
 
-static void fl_key_event_plugin_init(FlKeyEventPlugin* self) {}
+static void fl_key_event_plugin_init(FlKeyEventPlugin* self) {
+  self->pendingEvents.clear();
```

Comment:
Yeah, I added this when I had `pendingEvents` as a std::map, because for some reason it would throw if I tried to insert without clearing first.

I left this after going back to a deque because I wasn't sure if init could be called again or not. I'll remove it.